### ISSUE TITLE
About VPN's 

### ISSUE THREAD

**Issue by CHEF-KOCH** [CHEF-KOCH/NSABlocklist#21]
**This is a won't fix/close topic.**

Since I often get asked which VPN you _should_ use or why using a VPN at all and if there any news about VPN's I decided to open this issue ticket to keep you informed.

I often read something against VPN's and I like to comment on some things:

```bash
// Most arguments I read/hear from some people
1.) Never trust VPN providers, and avoid using them altogether
2.) As an alternative, consider using WireGuard for supporting critical corporate infrastructure
3.) For personal privacy and anonymity use Tor
4.) VPNs don’t offer the safety, privacy or, the level of encryption and anonymity which they commonly claim to. They simply can’t do so, speaking practically. Most of them not being open source makes matters even worse. Hence, their software’s code quality in terms of security is quite understandable, they simply try to hide this aspect.
5.) Don’t fall for advertising clickbait without further analyzing and doing personal research into them.
```

1.) Not everyone can or have the money/knowledge to build an on VPN service so the alternative is here to trust 'others'. Building your own VPN service might even end up with a lesser secure setup because you have to spend a lot of money/research to build a hardened infrastructure - using an external service might even be cheaper over the long run since you don't have to waste time/money in order to keep it secure. However, I do agree there is no trust in the internet but this is not an argument it's an opinion.

2.) WireGuard is still in beta and not much router firmware and VPN provider supporting it. WireGuard is a promising alternative to OpenVPN, the code is smaller which means it#s easier to audit, maintain and contribute but it takes more time and a final audit in order to recommend it. I see this on IM messenger a new "secure" alternative is popping up and it automatically gets a suggestion on e.g. PrivacyTools.io - this is seriously the wrong way, there has to be a professional audit + it needs to be tested and verified. The source code alone itself is no evidence that it's secure/insecure. 

3.) Tor is not the only network in the world, there is I2P and others. Outgoing encrypted traffic is fine but what about incoming one?! Right! be smart and combine them! This statement "use tor" is also dangerous because a normal user might use an insecure/weak configuration because he has no clue or he makes mistakes. 

4.) Here is an [overview of how a VPN works](https://www.infosec.gov.hk/english/technical/files/vpn.pdf) (by infosec). I suggest that this guy really reads it ... it might help to "get started".

5.) As a matter of fact, some VPN's block advertising (you can manually opt-in) which uses their own pre-made lists which blocks several domains. The only thing to complain here is that there is no way to contribute to such list or see them in public - you only can report blocked content. However, PIA and proton want to open source their clients in the near future. 

This article [here](https://latesthackingnews.com/2018/09/26/your-privacy-is-at-risk-from-using-a-vpn-an-interview-with-scott-arciszewski/) and this gist [here](https://gist.github.com/joepie91/5a9909939e6ce7d09e29) is wrong in so many ways written from people which don't even know what VPN is.

## Small overview how a VPN works 

A VPN (Virtual Private Network) works as follows: A program, or in some cases the operating system encrypts the packets to be sent, these packets would not be seen in a way that anyone can read, but they need the key, which is on the VPN server. This server decrypts the packets and sends them to the desired server, in this example a website. For the website, a computer is connecting, but it does not have the actual information of where it is receiving the connection, as it only detects the VPN server by sending the packets. The return process is the reverse of what has just happened.

## Strengths

As previously described, the VPN gives extra encryption to Internet traffic, even for a moment, this means that your Internet provider, malicious hackers and other risks will have difficulty accessing important information, such as passwords or bank details in purchases by the Internet. Another positive side is that by "disguising" your IP can access censored sites in certain countries.

## Weaknesses

Of course, no solution is perfect since they can all be weakened by a malicious hacker, with the VPN is the same, so you must keep in mind the following things: If the VPN server is compromised by a hacker, this person will be able to see those that pass through the server, so it is recommended only to trust companies that are rigorously testing their security.

If someone manages to have the keys to decrypt the packets, another person can connect between the VPN and you to analyze their movement, this can be mitigated locally with antivirus and other protection programs, while the VPN server will have to protect itself once again.

Some private network services will allow you to use their private servers in exchange for your data. Of course, they might not state their intention in the first place.

## Jurisdiction

The jurisdiction doesn't mateter as long as the encryption is strong enough, in order words even if the data are hand-over to CIA, KGB & Co. they would only see gibberish. Some jurisdiction also can be bypassed since the traffic goes trough countries which have other jurisdiction which then might forces them to give anyway the logs away (if there are any). 

It's more interessting and important than the log-file is scrambled/deleted/encrypted and that the server + client configuration has [zero-knowledge proof](https://en.wikipedia.org/wiki/Zero-knowledge_proof). 

## Additional comment

Despite not being a 100% sure way to keep your security online, there really is not any tool that is perfect, so you should do your best to get the most secure online.

It's true that VPN might have some weaknesses but this stands and falls with the encryption configuration (which can be reviewed or changed on the server/client-side).

Against VPN:
* https://latesthackingnews.com/2018/09/26/your-privacy-is-at-risk-from-using-a-vpn-an-interview-with-scott-arciszewski/
* https://gist.github.com/joepie91/5a9909939e6ce7d09e29
* https://gist.github.com/spaze/558b7c4cd81afa7c857381254ae7bd10
* https://gist.github.com/kennwhite/1f3bc4d889b02b35d8aa
* https://security.stackexchange.com/questions/86806/disadvantages-of-using-a-vpn
* https://www.theregister.co.uk/2016/02/26/ssl_vpns_survey/

VPN and what to know:
* https://www.privateinternetaccess.com/pages/tor-vpn-proxy
* https://news.ycombinator.com/item?id=16371030
* https://thatoneprivacysite.net/vpn-comparison-chart/
* https://news.ycombinator.com/item?id=16699270
* https://gist.github.com/CHEF-KOCH/3019362ffc8526d6717fb6d1cb9a025b
* https://github.com/lexrus/VPNOn
* https://github.com/Microsoft/PQCrypto-VPN
* https://github.com/OpenVPN
* https://www.wireguard.com ([Review](https://arstechnica.com/gadgets/2018/08/wireguard-vpn-review-fast-connections-amaze-but-windows-support-needs-to-happen/))

**Recommadation**
* ProtonVPN (already listed)
* PerfectPrivacy (already listed) 
* PIA  (already listed)

Keep in mind that I not add more VPN providers unless there is enough evidence that it's worth to get a serious recommadation. Everyone can claim xyz but to hold such a promise and to proove that are two different things.

**Comment by User486375** [comment]
I mostly agree with your first 4 points. Point 5 seems to miss the mark of your own question though.

> 5.) Don’t fall for advertising clickbait without further analyzing and doing personal research into them.

Misleading VPN advertisements is a huge problem, VPNs themselves blocking ads is completely irrelevant to this issue. To act like crappy "VPN Review sites" which don't give a shit about privacy and just want their affiliate clicks and literally rank VPNs as such. See [IVPNs reasoning for closing their affiliate program](https://www.ivpn.net/blog/closed-affiliate-program). And this is not to say there still can't be "good VPNs" I just think people should put some thought into picking one and not just pick the cheapest or most spam advertised VPN out there.

>Recommadation
ProtonVPN (already listed)
PerfectPrivacy (already listed)
PIA (already listed)
NordVPN (already listed)

What exactly is your reasoning for your VPN recommendations, I don't see what I'd consider the most privacy respecting VPNs, e.g. Mulvad, IVPN and AzireVPN to name a few. Also, NordVPN removal?

> Controversial:
Based on the following [review](https://docs.google.com/spreadsheets/d/e/2PACX-1vRh1eSvC9A9hvNE9m1ZgfZQu5GtREEXtKQ25BmCLveYduOl4kVc5gDO7Mj28oOboAv-VTIMtY7JdKpP/pubhtml#) these VPN services are marked as controversial.

This and your current list just seem pretty lazy to me for someone who bothered to make an NSA blocklist. I don't proclaim to be the sole arbiter of truth about what makes a good privacy respecting VPN. I just find it a stark contrast that you used a BestVPN comparison chart which listed NordVPN at the top based solely on number of servers and ability to unblock Netflix while ignoring things that actually matter when it comes to privacy, to somehow mark VPNs such as IVPN "Controversial" whatever that means. While actually respectable donation driven sites such as privacytools.io [has listed Mullvad, IVPN and ProtonVPN](https://www.privacytools.io/providers/vpn/) and thatoneprivacysite.net [which had reviewed NordVPN negatively](https://thatoneprivacysite.net/blog/nordvpn-review/) all the way back in 2016.

If you named your lists best VPNs to access Netflix, I'd understand it better.

>The jurisdiction doesn't mateter as long as the encryption is strong enough, in order words even if the data are hand-over to CIA, KGB & Co. they would only see gibberish. Some jurisdiction also can be bypassed since the traffic goes trough countries which have other jurisdiction which then might forces them to give anyway the logs away (if there are any).

Yes it does, please tell me how encryption is relevant whatsoever and I also don't get what you mean by "they only see gibberish". If data was logged it won't be logged and magically be encrypted and unable to be decrypted by the same people who encrypted it. Also, the concern isn't that a no logging VPN company will hand over logs, it's that they could depending on jurisdiction be forced to log users without their knowledge with a gag order, see the last controversy of Riseup listed [here](https://en.wikipedia.org/wiki/Riseup#Controversies).

It seems like you're answering this question in relation to mass surveillance but that isn't peoples issue with jurisdiction so it misses the point either way, though you literally say "if the data are hand-over".

> Against VPN:
https://www.theregister.co.uk/2016/02/26/ssl_vpns_survey/

This article  is referring to corporate SSL VPNs which let alone being irrelevant considering it's not referring to commercial VPNs, but they don't even use OpenVPN or Wireguard.

**Comment by CHEF-KOCH** [comment]
> What exactly is your reasoning for your VPN recommendations, I don't see what I'd consider the most privacy respecting VPNs, e.g. Mulvad, IVPN and AzireVPN to name a few. Also, NordVPN removal?

The original list was proven as "raid secure" which means FBI could not get any data from the extraction process, which is the proof that they hold their promise. However, much changed and I changed the list according to latest findings and reviews, tests and on how (overall) reliable the service is. NordVPN was removed because it got hacked (data leakage), of course they fixed it but I lost all trust in it. NordVPN is also a "new" player compared to the old ones like PIA and PP. But don't worry PIA got removed too because owner changed, so the "trust" argument does matter in a context that if a provider can't even hold their own promise, how secure they might actually be?! And a new owner or investor typically cares about money, not "security" in the first place (_that's what history showed us multiple times_).

> This and your current list just seem pretty lazy to me for someone who bothered to make an NSA blocklist. I don't proclaim to be the sole arbiter of truth about what makes a good privacy respecting VPN. I just find it a stark contrast that you used a BestVPN comparison chart which listed NordVPN at the top based solely on number of servers and ability to unblock Netflix while ignoring things that actually matter when it comes to privacy, to somehow mark VPNs such as IVPN "Controversial" whatever that means. While actually respectable donation driven sites such as privacytools.io has listed Mullvad, IVPN and ProtonVPN and thatoneprivacysite.net which had reviewed NordVPN negatively all the way back in 2016.

I basically agree, the "chart" was added (_I'm no friend of the chart btw._) because people keep asking me "what is a good VPN" - and the chart overall explains the things which are important to check (_for beginners_). It cannot be verified, I guess that is your point. But I do not want to talk about "can you trust xyz VPN provider". I typically test the VPN's which I list and there was never a problem with them nor reason to believe that they didn't hold their promise.

 > Yes it does, please tell me how encryption is relevant whatsoever and I also don't get what you mean by "they only see gibberish". If data was logged it won't be logged and magically be encrypted and unable to be decrypted by the same people who encrypted it. Also, the concern isn't that a no logging VPN company will hand over logs, it's that they could depending on jurisdiction be forced to log users without their knowledge with a gag order, see the last controversy of Riseup listed here.

If the encryption is strong enough than the attacker only see traffic which can't be decrypted (_this is gibberish text_). The assumption that some countries do not have an interest in the data traffic was spread by privacytools.io community without any evidence that e.g. country x is "not been monitored" just because the is no official program. Some organization working together across "rules" "jurisdiction" and they share data to catch "bad guys" (this was showed multiple times, congress lied, FBI and CIA lied etc). Interpol does work together with FBI and CIA, and Interpol build a group to fight "internet crime", so how naive someone seriously should be to not believe that they don't exchange their findings?! The only protection is strong encryption and that you delete your logs, that is preventing (in theory) anyone to reveal what content your traffic contains exactly. We are not even talking about the possibilities to re-route the traffic to other "nodes", like the Russians already did to obtain (possible obtain data _speculation_).

I basically automatically assume that there is interest in handing over data or to infiltrate bigger endpoints around the globe, which every traffic must go trough and this has nothing to do with jurisdiction, they just do it... 

> This article is referring to corporate SSL VPNs which let alone being irrelevant considering it's not referring to commercial VPNs...

It doesn't matter if it's commercial VPN or your own created VPN service, the point here is that due to misconfiguration or outdated configurations/certificates/protocols a VPN is not automatically more secure. 


>  which had reviewed NordVPN negatively all the way back in 2016.....

Yes, and since then things changed.


**About other communities**
**PrivacyTools and any other "privacy" focused community are in general not welcome in this project since they claim a lot without giving any specific proof, they even interpret NSA leaks wrong. For example, they assume (just because ONE picture) that Google, etc working together with NSA, this is not the case because Google is forced (by law) to do that, not because NSA "wants" it, that's a difference. Google (_in this example_) has no interest to hand over data because they are the ones which sell data for money and not for free. There is also the reputation at stake + the fact that it costs money (_time_) to prepare the data, exclude specific things etc. I doubt that an organization wants to do extra work for no money (_the leak not showed any evidence that NSA paid Google, MS & Co with money for data_).**

This project is not influenced by:
- Money
- Popularity
- DOES NOT interpret leaks wrongly like (most) NOT ALL of course, privacy focused communities (that is the difference)
- Does not recommend anything without tests (reference links are given and those links are not in relationship with this project since I can't control their content nor do they provide a change log (in most cases). However at time of posting it was checked and approved to make an overall point which makes sense to the subject.

**Comment by User486375** [comment]
Thanks for the detailed responses.

>The original list was proven as "raid secure" which means FBI could not get any data from the extraction process, which is the proof that they hold their promise. However, much changed and I changed the list according to latest findings and reviews, tests and on how (overall) reliable the service is. NordVPN was removed because it got hacked (data leakage), of course they fixed it but I lost all trust in it. NordVPN is also a "new" player compared to the old ones like PIA and PP. But don't worry PIA got removed too because owner changed, so the "trust" argument does matter in a context that if a provider can't even hold their own promise, how secure they might actually be?! And a new owner or investor typically cares about money, not "security" in the first place (that's what history showed us multiple times).

Source on a ProtonVPN or NordVPN raid? Considering you said you recommended only raid proof ones.

Why isn't Mullvad listed whatsoever?

Are you going to edit/update this issue also? 

Why isn't a VPN having things such as zero tracking on their site, Tor Addresses, Cash/Crypto Payment options, not misrepresenting VPNs in advertisements, spam them or allowing affiliates to do so willy nilly, taken into account at all when deciding if a VPN is trustworthy? Rather than the luck of being Raided, because it's not the case that every VPN provider who hasn't been raided is just a honeypot.

>I basically agree, the "chart" was added (I'm no friend of the chart btw.) because people keep asking me "what is a good VPN" - and the chart overall explains the things which are important to check (for beginners). It cannot be verified, I guess that is your point. But I do not want to talk about "can you trust xyz VPN provider". I typically test the VPN's which I list and there was never a problem with them nor reason to believe that they didn't hold their promise.

Then remove the chart rather than referencing it as a good guide to a trustworthy VPN, it doesn't mean you can't still link to the chart as a reference for info about VPNs, but saying "Based on the following [review](https://docs.google.com/spreadsheets/d/e/2PACX-1vRh1eSvC9A9hvNE9m1ZgfZQu5GtREEXtKQ25BmCLveYduOl4kVc5gDO7Mj28oOboAv-VTIMtY7JdKpP/pubhtml#) these VPN services are marked as controversial." is a ridiculous misrepresentation. It also still ranks things such as NordVPN and PIA highly and doesn't list breaches. Your list is you giving your opinion on what you think is best for what I assume is their privacy, not what appeals to the most amount of people possible.

>If the encryption is strong enough than the attacker only see traffic which can't be decrypted (this is gibberish text). The assumption that some countries do not have an interest in the data traffic was spread by privacytools.io community without any evidence that e.g. country x is "not been monitored" just because the is no official program. Some organization working together across "rules" "jurisdiction" and they share data to catch "bad guys" (this was showed multiple times, congress lied, FBI and CIA lied etc). Interpol does work together with FBI and CIA, and Interpol build a group to fight "internet crime", so how naive someone seriously should be to not believe that they don't exchange their findings?! The only protection is strong encryption and that you delete your logs, that is preventing (in theory) anyone to reveal what content your traffic contains exactly. We are not even talking about the possibilities to re-route the traffic to other "nodes", like the Russians already did to obtain (possible obtain data speculation).

I agree that the 5, 9, and 14 eyes assumption is stupid, as there's no reason to think every other country isn't doing the same or worse (another good reason to remove that Google doc as reasoning). But my point was that your message doesn't concisely convey that it's even referring to mass surveillance only. I think jurisdictions such as the U.S. especially and any other country with gag order equivalent laws should be taken into account when picking a VPN.

>About other communities
PrivacyTools and any other "privacy" focused community are in general not welcome in this project since they claim a lot without giving any specific proof, they even interpret NSA leaks wrong. For example, they assume (just because ONE picture) that Google, etc working together with NSA, this is not the case because Google is forced (by law) to do that, not because NSA "wants" it, that's a difference. Google (in this example) has no interest to hand over data because they are the ones which sell data for money and not for free. There is also the reputation at stake + the fact that it costs money (time) to prepare the data, exclude specific things etc. I doubt that an organization wants to do extra work for no money (the leak not showed any evidence that NSA paid Google, MS & Co with money for data).

Then why not try to convince them of your positions on their GitHub? Unless you think that's a waste of your own time, but they don't seem like unreasonable people to me.

**Comment by CHEF-KOCH** [comment]
> Source on a ProtonVPN or NordVPN raid? Considering you said you recommended only raid proof ones.

Proton has (most) or a lot of parts [open sourced](https://github.com/ProtonVPN) + an explained Blog post to almost every change they make. Mullvad is more or less yet another newcomer which I need to review more [since Mozilla is working together with them](https://mullvad.net/de/blog/2019/12/3/mullvad-partnerships-page-has-been-updated-mozilla/). I expect several changes once it's out of "Beta".

> Why isn't a VPN having things such as zero tracking on their site, Tor Addresses, Cash/Crypto Payment options, not misrepresenting VPNs in advertisements, spam them or allowing affiliates to do so willy nilly, taken into account at all when deciding if a VPN is trustworthy? Rather than the luck of being Raided, because it's not the case that every VPN provider who hasn't been raided is just a honeypot.

VPN's are listed _optionally_ in our readme.md because people asked me to do that. This is in general not substantial to FBI/CIA, NSA etc. I'm not going to change the list every time a new VPN provider is going to show up. It usually takes ~1 year until I make a decision because I actually test the providers myself and wait for reviews from other professional folks. I'm not sure about affiliates, that seems to trigger you really hard, personally I do not care if a provider sells his own kids to make some money, this is an ethical discussion which has no place here. I would not use such a provider but that doesn't mean the service is "bad" or that they're not going to hold their promise/claim. We have to differ between a) ethical reasons b) promises/verification and c) if what the provider claims is going to happen is really happen, which often can be done if the source code is available (e.g. if the provider open's up the backend  software, so you could actually test and check if his claim is true or not).

> ...."NordVPN and PIA highly and doesn't list breaches..."

True, on the other side, I did not contact the author to be honest to "correct" the VPN list, I think more people actually would benefit if the list on Google Docs gets an update. However I consider your proposal to remove or separate the VPN chart entirely. An GitHub ticket seems to be a better place to talk about "VPN findings".

>  I think jurisdictions such as the U.S. especially and any other country with gag order equivalent laws should be taken into account when picking a VPN.

There is simply no proof (_or not enough evidence_) for all of this, I do not operate this way. I dislike lots of things what's going on around the web and to stay on the topic, what several VPN providers doing, but I can't start a "smear campaign" because I do not like xyz or I want to force them to change something. I do enough behind the scene and I think people know that, but I also get older, my motivation is sinking each year and personally I'm done arguing and arguing for nothing or if I'm not 100% in control, because the fear is high that nothing gets changed which results in a huge frustration.

> Then why not try to convince them of your positions on their GitHub? 

I tried to convince hundreds of people about my point of view and it never worked out for me I lost my lifetime for basically nothing and you can't change people's mind because you "want to". You can try to give them "facts" but that also (often, not always) doesn't work, people often defending their "loved" product/provider at all cost. So I gave up on it. What people are doing instead if they don't like what you have to say they are a) creating fake stuff about you to damage your reputation b) ban/exclude you from their website/project or c) entirely ignore you. A good example is my concerns regarding cryptostorm, I opened a ticket [here](https://github.com/cryptostorm/sysctl.conf/issues/1) they entirely ignore me and "play dead". There are lots of other examples and I only mean well at the end but if you do not have enough audience, no one is going to listen what you have to say, I'm.... and always will be..... bad at marketing and social stuff because it's simply not my thing. However, I joined several organizations (EFF, ACLU) regarding "fighting for people and the web", but that typically makes you not more credible because everyone can join and "support" them (with donations, ads etc).


**Changing the VPN section**
I hear your loud and clean and I think about your proposal to change the VPN section/part in the readme.md but first I'm going to contact the author of the VPN review/overview page, because I believe that would be the right way to do, adding some cells for additional notes and findings seems the easiest way. If the author refuses I create my own list based on my research which will be put on GitHub or another page, so that everyone can submit pull requests, or comment on their findings.

**Comment by User486375** [comment]
>Proton has (most) or a lot of parts open sourced + an explained Blog post to almost every change they make. Mullvad is more or less yet another newcomer which I need to review more since Mozilla is working together with them. I expect several changes once it's out of "Beta".

Hope I'm not misunderstanding you but, Mullvad has existed for [10 years](https://mullvad.net/en/blog/2010/1/28/version-20-released/) and is actually [open source](https://github.com/mullvad) unlike you saying ProtonVPN is when [it's not](https://github.com/ProtonVPN). ProtonVPN launched [2 and half years ago](https://protonvpn.com/blog/free-vpn-service-launch/) and still isn't open source. They open sourced their Android client alone [8 days ago...](https://github.com/ProtonVPN/android-app) if that's what you mean. That's honestly as laughable as [PIA Finally open sourcing theirs](https://github.com/pia-foss/desktop) in reaction to the backlash against transfer of ownership despite saying they would open source it over a year and a half ago that they would release the source code for all their clients [***over the next 6 months***](https://www.privateinternetaccess.com/blog/2018/03/private-internet-access-goes-open-source/). And speaking of [blog posts explaining changes which date back to 2010](https://mullvad.net/en/blog/).

Also, you are clearly not even trying to be informed on the matter if you conflate Mullvad itself with their Partnership with Firefox which is in beta. I don't think you're trying to be bad faith or anything but it can come off like that if you have strong beliefs on stuff you aren't or don't try to be informed on. I'm not trying to personally attack you, it just annoys me that it seems like I'm not getting through to you on some things. And I guess I feel like I have to call you out on them.

>VPN's are listed optionally in our readme.md because people asked me to do that. This is in general not substantial to FBI/CIA, NSA etc. I'm not going to change the list every time a new VPN provider is going to show up. It usually takes ~1 year until I make a decision because I actually test the providers myself and wait for reviews from other professional folks.

You can't seriously act like it takes you a year to decide on a VPN while putting in this little initial effort. How could you actually think Mullvad is a new VPN unless you know nothing about the VPN space.

>I'm not sure about affiliates, that seems to trigger you really hard, personally I do not care if a provider sells his own kids to make some money, this is an ethical discussion which has no place here. I would not use such a provider but that doesn't mean the service is "bad" or that they're not going to hold their promise/claim. We have to differ between a) ethical reasons b) promises/verification and c) if what the provider claims is going to happen is really happen, which often can be done if the source code is available (e.g. if the provider open's up the backend software, so you could actually test and check if his claim is true or not).

You can think that, but it's wrong imo. To say peoples past actions have no impact on their trustworthiness isn't born out in how society is structured or in practice. It's not to say companies or people can never change. But a company putting profit motive leaps above actually trying to respect privacy should be an easy indicator to someone privacy oriented not to use it. Ethics do matter. In the same sense of how google is willing to so easily create a censored search engine to help China and increase their own profit. This wasn't their own government forcing them to do so.

>However I consider your proposal to remove or separate the VPN chart entirely. An GitHub ticket seems to be a better place to talk about "VPN findings".

Thanks for considering it. Also, the reason I respond to this issue rather than another one is people will end up reading this and be misinformed on some things, that's why I asked you to edit this also. I honestly hadn't even read your full VPN lists before I read this, I dislike your full list a lot more honestly.

>There is simply no proof (or not enough evidence) for all of this, I do not operate this way. I dislike lots of things what's going on around the web and to stay on the topic, what several VPN providers doing, but I can't start a "smear campaign" because I do not like xyz or I want to force them to change something.

But there is proof of gag orders happening with companies? I linked already above [Riseup's case](https://en.wikipedia.org/wiki/Riseup#Controversies) which hosts a VPN and Email service. Off the top of my head there's more popular cases like [Lavabit](https://en.wikipedia.org/wiki/Lavabit#Suspension_and_gag_order). You might say, oh VPN providers you trust will do the same as Lavabit, right? I disagree. Being raided in a country you aren't even based in, and turning out to have no logs, then just pulling out of said country. Isn't the same as being willing to permanently shut down your entire company to not comply with a gag order.

***Lavabit is believed to be the first technology firm that has chosen to suspend or shut down its operation rather than comply with an order from the United States government to reveal information or grant access to information.***

>I do enough behind the scene and I think people know that, but I also get older, my motivation is sinking each year and personally I'm done arguing and arguing for nothing or if I'm not 100% in control, because the fear is high that nothing gets changed which results in a huge frustration.

>I tried to convince hundreds of people about my point of view and it never worked out for me I lost my lifetime for basically nothing and you can't change people's mind because you "want to". You can try to give them "facts" but that also (often, not always) doesn't work, people often defending their "loved" product/provider at all cost. So I gave up on it.

I get that, you don't have to bother change any other communities opinions if you're not bothered. Though I guess I want to change yours. Because I don't have a problem with most things. Even though it might seem like I have loads of problems. I think we can have constructive disagreements even if we never end up fully agreeing on things. I also probably got a lot more time to waste, currently at least hah.

>Changing the VPN section

My main problem is that the Controversial section exists. You are grouping what I consider providers that are more secure than your recommended ones with Opera VPN and Hola VPN. I think you should take no longing audits into account, of course you might say they can't be 100%. Though even more so take into account independent security audits. [This article](https://thewirecutter.com/reviews/best-vpn-service/) does for example. I also think that article is one of the best mainstream ones and a good resource to link to.

**Comment by CHEF-KOCH** [comment]
* Mullvad was launched (back in 2009) that's correct but the current partners and protocols they use and work with are pretty new (Wireguard/Mozilla). In WireGuard's case it still has some [problems](https://restoreprivacy.com/wireguard/)
* [ProtonVPN has a transparency report](https://protonvpn.com/blog/transparency-report/), the open source status from OpenVPN team was explained over [here](https://www.reddit.com/r/ProtonMail/comments/8qruum/no_commitment_to_open_source/). The libs they use are [OpenPGPjs](https://github.com/openpgpjs/openpgpjs) among several other libs, they are already foss. If we're talking about the back-end then this is correct there is no source code.
* [Proton](https://protonvpn.com/about) exist longer than 2 years and their history goes back to CERN, the VPN if that's what you're talking about is from 2017, that's right.
* PIA's source code was already there before the new owner ship was transferred. Check the GitHub and Commit history. Parts of it were not from the beginning uploaded because they had to fix several things, or that#s what they said. It goes back to[2018](https://www.reddit.com/r/PrivateInternetAccess/comments/84pkja/private_internet_access_goes_open_source/).

I'm not exactly sure what you want, if you like NordVPN and Mullvad than use it, what#s the problem?! I do not recommend a service which offers "new" stuff when there are weaknesses and when it's unclear how the new partners influencing the service.

> You can't seriously act like it takes you a year to decide on a VPN while putting in this little initial effort. How could you actually think Mullvad is a new VPN unless you know nothing about the VPN space.

It took a half year until found out that WireGuard isn't secure as they claim it will be, of course it's fixable but you can't expect to review immediately every single commit and understand what's going on, this takes time and effort. MullVad is a new player with the features they used,  it was one of the first which had (only an example) WireGuard. You should also not forget that introducing the actual service and creating the trademark/cooperation are two different things.


> You can think that, but it's wrong imo. To say peoples past actions have no impact on their trustworthiness isn't born out in how society is structured or in practice. It's not to say companies or people can never change. But a company putting profit motive leaps above actually trying to respect privacy should be an easy indicator to someone privacy oriented not to use it. Ethics do matter. In the same sense of how google is willing to so easily create a censored search engine to help China and increase their own profit. This wasn't their own government forcing them to do so.

[Elon Musk smoked weed on TV](https://www.theverge.com/2018/11/29/18118769/elon-musk-smoke-weed-nasa-admin-jim-bridenstine-workplace-culture-review), you think I see him different now?! I don't, I respect him same like I did before. Who is going to say that ethics does matter when it comes to VPN's or trustworthiness? That's made up by PTIO community, and nothing else, people are about the service and not what the CEO or maybe ONE clown does, there is always black sheep and to "destroy" a whole cooperation because of actions from those people is more than unfair. 

> In the same sense of how google is willing to so easily create a censored search engine to help China and increase their own profit.

Your information regarding this are wrong, Google had to do it otherwise China said "[we are not going to work together with you](https://en.wikipedia.org/wiki/Dragonfly_(search_engine))". Providing a service and to get money for the work is IMHO not automatically bad, or you go to your chef tomorrow and say, "hey please don't give me money"? Google same like all others need money for their employees, for new innovations etc. I don't see there relationship here between money, VPN's, ethics and censorship. 

>  I dislike your full list a lot more honestly.

Everyone is welcome to create a pull request. 

> But there is proof of gag orders happening with companies? I linked already above Riseup's case which hosts a VPN and Email service.

Hosting something and providing a service to "defeat" surveillance are two different things. In [Lavabit's case it's not going to stop them from returning](https://en.wikipedia.org/wiki/Lavabit). [LavaBit printed out the crypto-keys](https://www.vice.com/en_us/article/gv5vxb/crypto-keys-printout-of-edward-snowdens-email-service-lavabit-ladar-levison) which is worthless. I don't see how your jurisdiction argument holds here. If illegal stuff is involved like money laundering, or other criminal activities no jurisdiction will prevent that you soon or later will get busted, even if you like you're super smart like [Agora](https://www.wired.com/story/dark-web-drug-takedowns-deepdotweb-rebound/), [Silk Road](https://en.wikipedia.org/wiki/Silk_Road) & Co. Your provided Wkipedia link ([point 22](https://www.zdnet.com/article/encrypted-email-provider-riseup-misses-warrant-canary-deadline/)) links to eMail, not VPN (VPN was not affected). The FBI got warrants because there was evidence enough that the eMail service was "abused" for Ransomware.

**Comment by CHEF-KOCH** [comment]
- [All ProtonVPN apps are now open source and audited](https://protonvpn.com/blog/open-source/).
- Contacted the VPN chart provider today.
- NordVPn contacted (today) for interview/statement about last leak and how they're going to prevent it in the future and what steps are already taken.
- Mullvad contacted (yesterday) to give us more details about current implementation and Mozilla cooperation/influence.

**Comment by CHEF-KOCH** [comment]
We got finally proof that supports my theory that agencies never give a shit about jurisdiction. Which means the "do not use server or VPN from 199 eyes" country is bullshit. 

Source:
* https://www.zdf.de/nachrichten/politik/cryptoleaks-bnd-cia-operation-rubikon-100.html#xtor=CS5-21
* https://www.nytimes.com/reuters/2020/02/11/world/europe/11reuters-swiss-cyber.html
* https://www.washingtonpost.com/graphics/2020/world/national-security/cia-crypto-encryption-machines-espionage/?itid=hp_hp-top-table-main_crypto-730am%3Ahomepage%2Fstory-ans
* https://www.srf.ch/play/tv/sendung/rundschau?id=49863a84-1ab7-4abb-8e69-d8e8bda6c989
